examples/token-auth-keycloak/conf/proxy/nginx.conf

```diff
@@ -1,15 +1,17 @@
 server {
     listen       80;
     server_name  localhost;
-    resolver 127.0.0.11 valid=30s;
+    resolver 127.0.0.11 valid=30s ipv6=off;
 
     set $keycloak "http://keycloak:8080";
     set $registry "http://registry:5000";
     set $ui "http://ui";
 
+    charset utf-8;
 
-    #charset koi8-r;
-    #access_log  /var/log/nginx/host.access.log  main;
+    rewrite_log on;
+    access_log /dev/stdout main;
+    error_log /dev/stdout debug;
```

Comment:
This is an example, please keep this simple and do not add unnecessary configuration

Reply:
This is configuration for debugging and see what is happen in logs, i think people want to see this output if they use this example as playground )

Reply:
Some of my users use the examples directly without modifying them. I don't want them to have a development setup in production. You can remove or comment it